resources/dicts/thoughts/alive/general.json

```diff
@@ -51,87 +49,1497 @@
+            "playful"
+        ],
+        "random_living_status": [
+            "living"
+        ]
+    },
+    {
+        "id": "rebellious_gen",
+        "thoughts": [
+            "Is finding ways to rebel against the instructions {PRONOUN/m_c/subject} were given",
+            "Is doubting why things are done the way they are",
+            "Taunted a dog stuck behind a Twoleg fence",
+            "Started an argument with the leader during a Clan meeting",
+            "Is spreading doubt through {PRONOUN/m_c/poss} Clanmates over the way things are done",
+            "Has simply decided not to do what {PRONOUN/m_c/subject} were told"
+        ],
+        "main_trait_constraint": [
+            "rebellious"
+        ]
+    },
+    {
+        "id": "responsible_gen",
+        "thoughts": [
+            "Is planning to wake up early tomorrow",
+            "Was the first to rise this morning",
+            "Reminds an apprentice to follow the code",
+            "Gently reminds a Clanmate of something they forgot about",
+            "Never forgets to thank StarClan for the food",
+            "Roused the other cats awake at dawn",
+            "Isn't sure {PRONOUN/m_c/subject} should be idling around at the moment",
+            "Is planning on going to sleep early tonight",
+            "Is making sure no work is being shirked!",
+            "Is offering to help Clanmates with their duties",
+            "Deeply apologizes for a mistake {PRONOUN/m_c/subject}{VERB/m_c/'ve/'s} made and tries to fix it",
+            "Goes to bed early so {PRONOUN/m_c/subject} can work more tomorrow"
+        ],
+        "main_trait_constraint": [
+            "responsible"
+        ]
+    },
+    {
+        "id": "responsible_to_app",
+        "thoughts": [
+            "Is reminding an apprentice to follow the code"
+        ],
+        "main_trait_constraint": [
+            "responsible"
+        ],
+        "random_status_constraint": [
+            "apprentice"
+            ]
+    },
+    {
+        "id": "rebellious_to_leadership",
+        "thoughts": [
+            "Had a lengthy debate with r_c",
+            "Rolled {PRONOUN/m_c/poss} eyes at a job r_c assigned {PRONOUN/m_c/object} to",
+            "Is being scolded for {PRONOUN/m_c/poss} rebellious actions"
+        ],
+        "main_trait_constraint": [
+            "rebellious"
+        ],
+        "random_status_constraint": [
+            "deputy",
+            "leader"
+            ]
+    },
+    {
+        "id": "responsible_kits",
+        "thoughts": [
+            "Is watching the kits to make sure none of them accidentally hurt each other",
+            "Is teaching r_c the importance of following the code"
+        ],
+        "main_trait_constraint": [
+            "responsible"
+        ],
+        "random_status_constraint":[
+            "kitten"
+        ],
+        "random_living_status":[
+            "living"
+        ]
+    },
+    {
+        "id": "righteous_warrior",
```

Comment:
We don't characterize "righteous" as always meaning "follows StarClan", rather it's a cat who has deeply held beliefs that they strongly believe everyone should hold to.  You could certainly have a righteous cat who thinks their belief in the Dark Forest is unquestionable.  Or a righteous cat who doesn't believe in SC or DF and instead has a strong moral code of their own that they hold to.  You could move some of the SC specific thoughts to a thought group that requires the Star skill for now.

Reply:
Same as what I said for faithful